### PR TITLE
ROC-2874: Create Claim Draft - Bug fix

### PR DESCRIPTION
**Pull request by jacmurphy**
### JIRA link

https://tools.hmcts.net/jira/browse/ROC-2874

### Change description

Fixed the bug which didn't let you create more then 1 claim at a time.

### CODE REVIEW

File: src/test/features/testing-support/create-claim-draft.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
+///<reference path="../../../main/app/drafts/draft-data/claimDraft.ts"/>
```
Comment (geek-soft) [inline src/test/features/testing-support/create-claim-draft.ts]: Please remove

File: src/test/features/testing-support/create-claim-draft.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+  describe('on POST', () => {
+    checkAuthorizationGuards(app, 'post', pagePath)
+
+    context('when user authorised', () => {
+      beforeEach(() => {
+        idamServiceMock.resolveRetrieveUserFor('100', 'citizen')
+      })
+
+      it('should return 500 and render error page when cannot retrieve claim', async () => {
```
Comment (geek-soft) [inline src/test/features/testing-support/create-claim-draft.ts]: Please say `retrieve claim draft`, you are not retrieving claim at that point.

Comment (jacmurphy) [reply src/test/features/testing-support/create-claim-draft.ts]: 👍

File: src/test/features/testing-support/create-claim-draft.ts (hunk 1 above)
Comment (jacmurphy) [reply src/test/features/testing-support/create-claim-draft.ts]: 👍

File: src/main/app/drafts/draft-data/claimDraft.ts [hunk 3]
```diff
@@ -2,69 +2,71 @@ import { YesNoOption } from 'models/yesNoOption'
...
-        city: 'London',
-        postcode: 'SW1A 2AA'
-      },
-      dateOfBirth: {
-        known: true,
-        date: {
-          year: 1997,
-          month: 5,
-          day: 27
+export function createClaimDraft () {
```
[the remark this answers is not in the document]
Comment (geek-soft) [reply src/main/app/drafts/draft-data/claimDraft.ts]: Happy with prepare.

**Comment by geek-soft** [comment]
I tested it locally, it seems to work fine now.

**Pull request merged**